src/renderers/common/CubeRenderTarget.js

```diff
@@ -54,8 +54,14 @@ class CubeRenderTarget extends WebGLCubeRenderTarget {
 		if ( texture.minFilter === LinearMipmapLinearFilter ) texture.minFilter = LinearFilter;
 
 		const camera = new CubeCamera( 1, 10, this );
+
+		const currentMRT = renderer.getMRT();
+		renderer.setMRT( null );
+
 		camera.update( renderer, scene );
```

Comment:
We have renderer state definition and restoration in many places, I was thinking about having a way to automate this otherwise every new feature related will break the chain of nodes that use `beforeUpdate()` to `render()`.